### ISSUE TITLE
How do I update the padding of an ui.layout_sidebar

### ISSUE THREAD

**Issue by vindevoy** [posit-dev/py-shiny#1614]
This is my code:

```
from pathlib import Path

from shiny import App, ui, render, reactive

left_sidebar = ui.sidebar("Left Sidebar", id="left_sidebar", bg="#f8f8f8")
right_sidebar = ui.sidebar("Right Sidebar", id="right_sidebar", bg="#f8f8f8", position="right")
main_content = ui.output_ui(id="render_main")
right_part = ui.layout_sidebar(
                    right_sidebar, main_content, padding=[20, 50, 20, 50], id="right_part", border=False)

app_ui = ui.page_fillable(
            ui.layout_sidebar(
                left_sidebar,
                right_part,
                padding=0,
                id="left_sidebar_holder",
                border=False,
                border_radius=False
            ),
            id="page_main",
            title="Shiny Pivot",
            padding=0,
        )


def server(inputs, outputs, session):
    assert inputs is not None
    assert outputs is not None
    assert session is not None

    @render.ui
    def render_main():
        card = ui.card(
            ui.h1("Reporting"),
            ui.p(f"left_sidebar: {inputs.left_sidebar()}"),
            ui.p(f"right_sidebar: {inputs.right_sidebar()}"),
        )

        return card


app = App(app_ui, server, static_assets=Path(__file__).parent.parent.joinpath("pivot", "static"))

```

I want the padding of my layout_sidebar to be dynamic.

In case the left sidebar is closed, the 50 on the left can also become 20.  Same with the right sidebar.  
So far, I have not been able to update "properties" like this.  And I don't see it in the documentation.

**Comment by gadenbuie** [comment]
> In case the left sidebar is closed, the 50 on the left can also become 20. Same with the right sidebar.
> So far, I have not been able to update "properties" like this. And I don't see it in the documentation.

Let me confirm that I've understood you. You currently have `50px` of padding on the right and left sides of the `main_content` area in the `right_part` sidebar layout. When either sidebar is collapsed, you'd like to have only `20px` of space between the main content area and the collapsed sidebar. Is this what you're wanting to do?

If so, you're right that we don't expose parameters or settings that cover this situation. We assume that the padding around the main content area will be constant. You can certainly do this with CSS though.

That said, I'd caution against writing custom rules for this scenario. We rely on the padding in the left and right margins to provide space for the sidebar collapse toggle; setting the left padding to 20px when the left sidebar is collapsed	will end up causing the collapse toggle to overlap the content in the main area

![image](https://github.com/user-attachments/assets/9eb2e54f-1c8c-4b33-a604-4db3310aabd9)

Perhaps you could describe more what you're wanting to achieve with your app and why its important to change the padding in your app in this way?

**Comment by vindevoy** [comment]
No, vise versa.  When the sidebar is visible, I want the padding on that side to be 20px.  There's no need to lose 50px.  

When the sidebar is collapsed, it must be changed to 50px because otherwise the main content comes over the arrow to expand the sidebar again.  Which you described above.
